TestResultSummaryService/perf/DataManagerAggregate.js

```diff
@@ -5,11 +5,12 @@ const ObjectID = require( 'mongodb' ).ObjectID;
 class DataManagerAggregate {
     static aggDataCollect(childBuild) {
         const benchmarkMetricsCollection = {};
-        let name, variant, jdkBuildDate;
+        let name, variant, jdkBuildDate, version;
```

Comment:
Let's call it something like `javaVersion` to make it more clear.

Reply:
Currently, each test stored JDK date and version, something that isn't needed since JDK doesn't change in a Jenkins job even though we might be running different tests.

Please put a TODO here that we should move JDK data and version at the top level of `testResults` so that it can be used directly from there instead of having them at multiple places, unnecessary increasing the storage size of each `testResults` object.

Reply:
This TODO has been completed in this PR.